test/TestProjects/MgmtKeyvault/tests/Generated/Mock/SubscriptionExtensionsTest.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Net;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.TestFramework;
+using MgmtKeyvault.Models;
+
+namespace MgmtKeyvault.Tests.Mock
+{
+    /// <summary> Test for Subscription. </summary>
+    public partial class SubscriptionExtensionsMockTests : MockTestBase
```

Comment:
since we now merge all the extension classes into one big class, we might need to change the name here